pyproject.toml

```diff
@@ -59,9 +59,7 @@ tests = [
 
 [project.entry-points."node_graph.node"]
 "node_graph.node" = "node_graph.node:Node"
-"node_graph.graph_inputs" = "node_graph.nodes.builtins:GraphInputs"
```

Comment:
No need to hard-code a Node for each graph-level io: inputs, outputs, ctx. Now all use `GraphLevelNode`.